diff --git a/node_modules/@types/babel__traverse/index.d.ts b/node_modules/@types/babel__traverse/index.d.ts
index ab63f9d..11bed48 100644
--- a/node_modules/@types/babel__traverse/index.d.ts
+++ b/node_modules/@types/babel__traverse/index.d.ts
@@ -108,6 +108,8 @@ export class Scope {
 
     checkBlockScopedCollisions(local: Node, kind: string, name: string, id: object): void;
 
+    crawl(): void;
+
     rename(oldName: string, newName?: string, block?: Node): void;
 
     dump(): void;
@@ -120,7 +122,7 @@ export class Scope {
 
     registerConstantViolation(path: NodePath): void;
 
-    registerBinding(kind: string, path: NodePath, bindingPath?: NodePath): void;
+    registerBinding<T>(kind: string, path: NodePath<T>, bindingPath?: NodePath<T>): void;
 
     addGlobal(node: Node): void;
 
@@ -538,6 +540,8 @@ export class NodePath<T = Node> {
     getCompletionRecords(): NodePath[];
 
     getSibling(key: string | number): NodePath;
+    getPrevSibling(): NodePath;
+    getNextSibling(): NodePath;
     getAllPrevSiblings(): NodePath[];
     getAllNextSiblings(): NodePath[];
 
@@ -549,7 +553,7 @@ export class NodePath<T = Node> {
         : T[K] extends Node | null | undefined
         ? NodePath<T[K]>
         : never;
-    get(key: string, context?: boolean | TraversalContext): NodePath | NodePath[];
+    // get(key: string, context?: boolean | TraversalContext): NodePath | NodePath[];
 
     getBindingIdentifiers(duplicates?: boolean): Node[];
 
